src/doblib/action.py

```diff
@@ -346,8 +384,8 @@ def _action_update(self, env, model, domain, item):
             if name not in records._fields:
                 continue
 
-            if isinstance(apply_act, dict):
-                dynamic[name] = apply_act
+            if isinstance(apply_act, dict) or records._fields[name].type == "many2many":
+                dynamic[name] = apply_act if apply_act is not None else {}
```

Comment:
Why is many2many special here? What do you want to fix with that?

Reply:
Well ok because the empty value for Many2many would be `[]` or maybe even `[[5]]`. But I could specify constant values `[[5]]` or `[[4, "$ref$"]]` would be valid but would be handled dynamically here.

It wouldn't cause a problem when you specify it with `{}` if you don't want to set a parameter. Maybe the configuration is just wrong at this point. I mean you can also try to write a string into an Integer field with configuration but the problem is in the config.